showcase.json

```diff
@@ -3228,6 +3228,26 @@
   "twitter": "https://twitter.com/ICLighthouse",
   "youtube": "https://youtu.be/Z92o1oCH598",
   "submittableId": "41203767"
+},
+{
+  "name": "Ping",
+  "description": "Ping is a notification service that boosts ecosystem engagement by creating a direct communication channel between dapp developers and their respective users.",
+  "website": "https://h7jna-pqaaa-aaaak-afgiq-cai.icp0.io/",
```

Comment:
@Saumay-Agrawal Can you please let us know if Ping is live yet?

Reply:
@jennifertrin yes, the application is live on ICP mainnet at this link. Let me know if it isn't working for you. I'll look into it.

Reply:
> @jennifertrin yes, the application is live on ICP mainnet at this link. Let me know if it isn't working for you. I'll look into it.

It takes about a minute to login. Are you running any other functions when a user logs in that is causing the slow speed?